data/common.yaml

```diff
@@ -32,8 +32,10 @@ common::author_host: localhost
 common::author_port: 443
 common::author_secure: 1
 
-common::dispatcher_conf_dir: /etc/httpd/conf.d/
 common::apache_conf_dir: /etc/httpd/conf
+common::dispatcher_conf_dir: /etc/httpd/conf.modules.d
+common::virtual_hosts_dir: /etc/httpd/conf.d
+common::ssl_cert: /etc/ssl/aem.unified-dispatcher.cert
```

Comment:
Please rename `common::ssl_cert` to `common::dispatcher_ssl_cert` since it's not applicable to other components, e.g. it's not a common location for SSL cert for orchestrator and chaos monkey components.

Downstream variables can still be called `ssl_cert` since it's mostly used in dispatcher scenarios.

Reply:
Done.